apps/aecore/src/aec_peers.erl

```diff
@@ -80,6 +80,7 @@
 -define(DEFAULT_RESOLVE_MAX_RETRY,             7).
 -define(DEFAULT_RESOLVE_BACKOFF_TIMES,
         [5000, 15000, 30000, 60000, 120000, 300000, 600000]).
+-define(LOG_PEER_COUNT_TIMEOUT,             5000).
```

Comment:
I think this should be configurable?! 🤔

Reply:
Everything in sync is configurable, so I made this interval configurable too.